common/service/dynamicconfig/constants.go

```diff
@@ -197,6 +197,7 @@ var keys = map[Key]string{
 	WorkerTimeLimitPerArchivalIteration:             "worker.TimeLimitPerArchivalIteration",
 	WorkerThrottledLogRPS:                           "worker.throttledLogRPS",
 	ScannerPersistenceMaxQPS:                        "worker.scannerPersistenceMaxQPS",
+	EnableBatcher:                                   "worker.enableBatcher",
```

Comment:
plz move this to section `// system settings`, around line 53

Reply:
Sure